### PR TITLE
Added "SpringOne Platform - A birds-eye view" post

### PR DESCRIPTION
**Pull request by TYsewyn**

### CODE REVIEW

File: _posts/2018-09-27-SpringOne-Platform.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+That's why Pivotal has build a tool to help teams articulate and align around business outcomes to give organizations the high-level view so they can assure that everyone is aligned and delivering value.
+
+Watch the talk <a href="https://www.youtube.com/watch?v=1ezNJvajUU0&feature=youtu.be&t=11m43s" target="_blank">here</a>.
+
+### Pivotal Act
+
+> "Pivotal Act is a program that partners with humanitarian organizations and charities to identify, design, and develop practical solutions to pressing challenges around the world."
+
+Together with developers, designers, and engineers from Pivotal Labs, Pivotal is going to apply the same methodology like they use in other engagements with clients but tailored to the needs of the humanitarian and social impact sectors instead of just donating technology or funds.
```
Comment (pvanhees) [inline _posts/2018-09-27-SpringOne-Platform.md]: Pivotal is going to apply the same methodology ~~they~~ use in other engagements with clients **,** but tailored

File: _posts/2018-09-27-SpringOne-Platform.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+This way charitable and nonprofit organisations can use the present-day technologies and put them directly to use through the partnership with Pivotal, while building up the organisation's technology capabilities so they can continue their work after the engagement end.
```
Comment (pvanhees) [inline _posts/2018-09-27-SpringOne-Platform.md]: the engagement end**s**

File: _posts/2018-09-27-SpringOne-Platform.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+When using the reactive model, people were limited to NoSQL databases because those were the only ones that had a driver which supports things like streams and backpressure.
+During the keynote Oliver Gierke announced the R2DBC project.
+This project currently consists out of a client, an SPI and a PostgreSQL implementation to bring the reactive capabilities to SQL databases.
+It also has an adapter to support the OpenJDK incubator project "ADBA", a non-blocking database access API that Oracle is proposing as a Java standard.
+
+### RSocket
+
+On stage Stéphane Maldini announced the RSocket project.
+RSocket is a binary application protocol that provides reactive streams semantics which is built by Facebook, Netifi and Pivotal.
```
Comment (pvanhees) [inline _posts/2018-09-27-SpringOne-Platform.md]: RSocket, built by Facebook, Netifi and Pivotal, is a binary....

File: _posts/2018-09-27-SpringOne-Platform.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+This protocol is payload, transport AND language agnostic making it easy to send eg. JSON or protobuf payloads over TCP, UDP, WebSockets or HTTP/2 using Java, Kotlin, C++, JavaScript,...
```
Comment (pvanhees) [inline _posts/2018-09-27-SpringOne-Platform.md]: This protocol is payload-, transport- AND language agnostic

File: _posts/2018-09-27-SpringOne-Platform.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+The highlights of this release are the support for JDK 11, initial refinements for GraalVM compatibility and Reactor & Hibernate got an upgrade to respectively Californium and 5.3.
+Improved startup times and less heap memory consumption are also some benefits you get when upgrading to this release.
+
+Check out <a href="https://spring.io/blog/2018/09/21/spring-framework-5-1-goes-ga" target="_blank">this post</a> by Juergen Hoeller to find out more!
+
+### Spring Boot 2.1.0.M4
+
+The fourth milestone of Spring Boot 2.1 got released to incorporate the 5.1 release of the Spring framework and closes over 40 issues and pull requests.
+
+Madhura Bhave wrote <a href="https://spring.io/blog/2018/09/25/spring-boot-2-1-m4-available-now" target="_blank">this post</a> announcing the milestone release and added some usefull links to the release notes and updated reference documentation.
```
Comment (pvanhees) [inline _posts/2018-09-27-SpringOne-Platform.md]: useful~~l~~

File: _posts/2018-09-27-SpringOne-Platform.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+
+Watch the talk <a href="https://www.youtube.com/watch?v=1ezNJvajUU0&feature=youtu.be&t=10m39s" target="_blank">here</a>.
+
+### Pivotal Tracker: Maestro
+
+> What happens when your organization scales, and you end up with lots of fast-moving developers?
+> It's really, really easy to get misaligned.
+> And when you get misaligned, it's really easy to ship the wrong thing.
+
+That's why Pivotal has build a tool to help teams articulate and align around business outcomes to give organizations the high-level view so they can assure that everyone is aligned and delivering value.
```
Comment (Turbots) [inline _posts/2018-09-27-SpringOne-Platform.md]: has builT a tool

File: _posts/2018-09-27-SpringOne-Platform.md (hunk 5 above)
Comment (Turbots) [inline _posts/2018-09-27-SpringOne-Platform.md]: usefuL (1 L)

File: _posts/2018-09-27-SpringOne-Platform.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+To learn more about this interesting project, visit this <a href="https://github.com/spring-projects/spring-fu" target="_blank">link</a>!
+
+### Spring Tools 4
+
+Spring Tools 4 is completely re-built from scatch after a decade of updates and improvements of Spring Tool Suite (STS).
+It's a new set of IDE agnostic tools that can be installed in your favorite IDEs and editors.
+Currently they support Eclipse, Visual Studio Code and Atom IDE.
+With this release also comes the end of Spring Tools 3, but not until mid 2019!
+STS 3.9.x will still receive updates and will be shipped as a full distribution, and the distribution will be updated to the upcoming Eclipse releases (2018-09, 2018-12, and beyond).
+After mid 2019, it will not receive any maintenance updates anymore.
```
Comment (Turbots) [inline _posts/2018-09-27-SpringOne-Platform.md]: it will no longer receive any maintenance updates

File: _posts/2018-09-27-SpringOne-Platform.md [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+authors: [tim_ysewyn]
+title: 'SpringOne Platform - A birds-eye view'
+image: /img/2018-09-27-SpringOne-Platform/post-image.jpg
+tags: [SpringOne, S1P]
+category: Spring
+comments: true
+---
+
+This year we went again to SpringOne Platform to check out the latest changes and what's to come.
+In this post I'll try to give an overview of everything that's just been released or coming in the future.
```
Comment (tomvdbulck) [inline _posts/2018-09-27-SpringOne-Platform.md]: coming in the future => coming in the near future

File: _posts/2018-09-27-SpringOne-Platform.md [hunk 9]
```diff
@@ -0,0 +1,151 @@
...
+
+Because it can be daunting for people to enter a community they are creating "first-timers-only" issues on the Spring Boot project, making it easier for newcomers to get connected.
+While working on these issues developers of the Spring Boot project will come alongside you to teach you to contribute back, help you and coach you.
+This way you will get to become part of the Spring community.
+
+Watch the talk <a href="https://www.youtube.com/watch?v=1ezNJvajUU0&feature=youtu.be&t=10m39s" target="_blank">here</a>.
+
+### Pivotal Tracker: Maestro
+
+> What happens when your organization scales, and you end up with lots of fast-moving developers?
```
Comment (tomvdbulck) [inline _posts/2018-09-27-SpringOne-Platform.md]: remove the ,

File: _posts/2018-09-27-SpringOne-Platform.md [hunk 10]
```diff
@@ -0,0 +1,151 @@
...
+> It's really, really easy to get misaligned.
+> And when you get misaligned, it's really easy to ship the wrong thing.
+
+That's why Pivotal has built a tool to help teams articulate and align around business outcomes to give organizations the high-level view so they can assure that everyone is aligned and delivering value.
+
+Watch the talk <a href="https://www.youtube.com/watch?v=1ezNJvajUU0&feature=youtu.be&t=11m43s" target="_blank">here</a>.
+
+### Pivotal Act
+
+> "Pivotal Act is a program that partners with humanitarian organizations and charities to identify, design, and develop practical solutions to pressing challenges around the world."
```
Comment (tomvdbulck) [inline _posts/2018-09-27-SpringOne-Platform.md]: , and => and

File: _posts/2018-09-27-SpringOne-Platform.md [hunk 11]
```diff
@@ -0,0 +1,151 @@
...
+If you want to find out more, just visit <a href="https://pivotal.io/act" target="_blank">https://pivotal.io/act</a>.
+
+You can watch the announcement <a href="https://pivotal.io/act" target="_blank">here</a>.
+
+### R2DBC
+
+When using the reactive model, people were limited to NoSQL databases because those were the only ones that had a driver which supports things like streams and backpressure.
+During the keynote Oliver Gierke announced the R2DBC project.
+This project currently consists out of a client, an SPI and a PostgreSQL implementation to bring the reactive capabilities to SQL databases.
+It also has an adapter to support the OpenJDK incubator project "ADBA", a non-blocking database access API that Oracle is proposing as a Java standard.
```
Comment (tomvdbulck) [inline _posts/2018-09-27-SpringOne-Platform.md]: Maybe also state that it is not yet production ready, but promising
see https://twitter.com/nebhale/status/1044962921577697280

File: _posts/2018-09-27-SpringOne-Platform.md [hunk 12]
```diff
@@ -0,0 +1,151 @@
...
+- Support for MongoDB 4.0 Client Sessions and Transactions
+- New Spring Data JDBC module
+- Apache Cassandra mapping improvements for Map and tuple types, Lifecycle Callbacks, and Kotlin Extensions
+- Replica Reads with Spring Data Redis
+
+To see what else has changed, check out <a href="https://spring.io/blog/2018/09/21/spring-data-lovelace-ga-released" target="_blank">this post</a> by Mark Paluch.
+
+### Spring Security 5.1
+
+In this new release a lot has been added regarding WebFlux support: OAuth 2, CORS and HTTPS redirection  are just a few to sum up.
```
Comment (tomvdbulck) [inline _posts/2018-09-27-SpringOne-Platform.md]: redirection  are => remove a space

**Pull request merged**